docs/index.rst

```diff
@@ -6,7 +6,37 @@ neXtSIM_DG : the next generation sea-ice model with a Discontinuous Galerkin met
 Introduction
 ------------
 
-`neXtSIM_DG` is a C++ model the simulates the sea ice ...
+**neXtSIM is a Lagrangian dynamical-thermodynamical sea-ice model that faithfully represents large-scale sea ice thickness, concentration and drift patterns** [1]_ [2]_ **, statistical properties of pack ice deformation and drift** [3]_ [2]_ **, diffusion and dispersion** [4]_ [5]_ **; and additionally exhibits a strong capacity for short-term sea ice forecasting at Pan-Arctic scales** [6]_ **. neXtSIM was developed to explore the role of sea-ice mechanics and dynamics in determining the large-scale behaviour of Arctic sea ice. As such, it has served as a testing and development ground for the brittle sea-ice rheologies developed over the past decade** [7]_ [8]_ [9]_ [2]_ **.** The initial development of the model was focused on understanding and representing as well as possible the impact of the new physics developments, which was the main rationale for the Lagrangian advection scheme used. As the new physics developments matured it became evident that these could impact not only the ice, but also the ocean and atmosphere below and above it [10]_.
```

Comment:
Starting the sentence, paragraph and section with an unexpanded, uncapitalized acronym looks really odd. Perhaps 'The neXt generation Sea Ice Model (neXtSIM) is a Lagrangian…'

Reply:
Yes I agree, I've updated it. If the next release is near, no need to do one for this then ! I just will make sure these changes are taken by merging it to main.